frontend/src/scenes/session-recordings/player/controller/PlayerSeekbarPreview.tsx

```diff
@@ -68,6 +72,10 @@ export function PlayerSeekbarPreview({ minMs, maxMs, seekBarRef }: PlayerSeekbar
 
     const isHovering = useIsHovering(seekBarRef)
 
+    const { featureFlags } = useValues(featureFlagLogic)
+    const alwaysShowSeekbarPreview = featureFlags[FEATURE_FLAGS.ALWAYS_SHOW_SEEKBAR_PREVIEW]
+    const canShowPreview = alwaysShowSeekbarPreview || maxMs < TWENTY_MINUTES_IN_MS
```

Comment:
Would it be possible to do this based off of active time instead? Might be nitpicking, but I'm thinking a 1 hour recording with only 5 mins of active time will still be easily previewable (probably where the preview is super useful as well)

Reply:
Good call... which also led me to a `TODO` in the metadata endpoint that reminded me we didn't actually return all of the meta data 🙈 

(I'm expecting the tests to fail, I can't run them locally because I've been testing Django 4.1 upgrade 🙃)